build.gradle

```
@@ -57,3 +57,7 @@ publishing {
         }
     }
 }
+
+wrapper {
```

Good ol' lazy task configuration APIs, my favorite:
```suggestion
tasks.named('wrapper', Wrapper) {
```